modules/migration/migration-resources/5.6.0/dbscripts/step1/identity/h2.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE IDN_OAUTH_CONSUMER_APPS ADD ID_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600000;
+
+CREATE TABLE IDN_AUTH_SESSION_STORE_TEMP (
```

Comment:
As per an internal discussion, the table name would be changed to IDN_AUTH_TEMP_SESSION_DATA_STORE. The scripts should be changed.

Also a new index is added to IDN_AUTH_SESSION_STORE table. Similar index should be used in TEMP table as well. Contact @rushmin on this matter.

Reply:
I will contact with @rushmin  and update the script